web/src/app/page.js

```
@@ -1,153 +1,482 @@
-                  {activeSection}
-                </h2>
-                <p className="text-gray-600 mt-1">
-                  Browse and manage your {activeSection} files
-                </p>
+              <div className="border-b border-gray-200 px-6 py-5 flex flex-col md:flex-row md:items-center md:justify-between gap-3">
+                <div>
+                  <h2 className="text-xl font-semibold text-gray-800">{sectionDetails[activeSection].title}</h2>
+                  <p className="text-gray-600 mt-1">{sectionDetails[activeSection].description}</p>
+                </div>
+                <div className="flex gap-2">
+                  <button className="rounded-md border border-indigo-200 bg-indigo-50 px-3 py-2 text-sm font-medium text-indigo-700 hover:bg-indigo-100">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)